README.md

```diff
@@ -0,0 +1,95 @@
+# 1. Introduction
+
+This repository is a library for implementing LwDP data source.
+
+The library used to analyze stream data using _aggregate operations_ mainly from
+the ["Lightweight Data Provider"](https://github.com/th2-net/th2-lw-data-provider). Data Services allows you to manipulate
+the stream data processing workflow using _pipelining_.
+
+The library allows you:
+
+- Natively connect to ["Lightweight Data Provider"](https://github.com/th2-net/th2-lw-data-provider) via
+  `GRPCDataSource` class and extract TH2 Events or Messages via _commands_
+- Build Event Trees (`EventsTreeCollection` class)
+
+# 2. Getting started
+
+## 2.1. Installation
+
+- From PyPI (pip)   
+  This package can be found on [PyPI](https://pypi.org/project/th2-data-services-lwdp/ "th2-data-services-lwdp").
+    ```
+    pip install th2-data-services-lwdp
+    ```
+
+- From Source
+    ```
+    git clone https://github.com/th2-net/th2-ds-source-lwdp
+    pip install th2-data-services-lwdp
+    ```
+
+## 2.2. Releases
+
+Currently there is only ds-lwdp v1 under developement.
+
+Newer releases will have separate branches indicated by SourceVersion of branch name.
+
+# 2.3 Versioning
+
+Versioning of the library looks like this:
+
+SourceVersion.Major.Minor.Patch
```

Comment:
What do you think if we update to?

DataSourceMajorVersion.LibVerison

`DataSourceMajorVersion` - the major version of LwDP
`LibVerison` - the version of the implementation in semantic versioning style Major.Minor.Patch

E.g. v1.0.1.0 - the data source implementation module for LwDP v1.x.y. Module version 0.1.0.

Reply:
![image](https://user-images.githubusercontent.com/25672975/196344296-92117cb7-18d9-420d-a436-7ac0126cb520.png)

1. Add example string like in my proposal above.
2. Let's wrap keywords to quotes like in my proposal above.
3. As you can see, there is no `/n` before `LibVersion` word. Be careful, please. Check your work before send it for review.